src/Avalonia.Controls/Presenters/TextPresenter.cs

```diff
@@ -17,6 +17,9 @@ namespace Avalonia.Controls.Presenters
 {
     public class TextPresenter : Control
     {
+        public static readonly StyledProperty<bool> ShowSelectionHighlightProperty =
```

Comment:
Shouldn't this property be added to the switch in `OnPropertyChanged` that triggers the call to `InvalidTextLayout`?